### DISCUSSION TITLE
Getting problems trying to query Views from Glue

### DISCUSSION THREAD

**Discussion by CamposIgor97** [trinodb/trino#9033] [Q&A]
I'm getting the following error when trying to query a storaged view on glue:
Query 20210830_121836_00003_v7ahb failed: line 1:15: Failed analyzing stored view 'hive.*SCHEMA*.*TABLE*': line 47:9: Catalog 'awsdatacatalog' does not exist
On the given line 47:9 I have a sub select referencing to another SCHEMA something like this: 
SELECT * FROM
      (
       SELECT * , "row_number"() OVER (PARTITION BY id ORDER BY migrated_at ASC, OP DESC) rank
       FROM
         *ANOTHER_SCHEMA*.*TABLE*
        )  ranked
WHERE ((ranked.rank = 1) AND OP <> 'D'))

Seems like the subselect is pointing to another catalog but when I check it with show create view ... its normal and I can even copy the SQL and paste it and it works.
 
Also I have tried with all the combinations of hive.translate-hive-views and hive.legacy-hive-view-translation.
Can somebody help me ? Anything I can try ?

**Comment by udbahv-agarwal-kb** [comment]
I am also facing the same issue. I am using trino with EMR along with glue as a catalog. I am trying to run query on some of the preexisting views in glue

**Comment by findinpath** [comment]
There is no support for reading from Athena views at the time of this writing in Trino
